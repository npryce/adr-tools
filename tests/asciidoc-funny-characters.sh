echo "adoc" > .adr-file-type
adr new Something About Node.JS
adr new Slash/Slash/Slash/
adr new -- "-Bar-"
ls doc/adr
