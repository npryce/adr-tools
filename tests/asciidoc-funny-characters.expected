echo "adoc" > .adr-file-type
adr new Something About Node.JS
doc/adr/0001-something-about-node-js.adoc
adr new Slash/Slash/Slash/
doc/adr/0002-slash-slash-slash.adoc
adr new -- "-Bar-"
doc/adr/0003-bar.adoc
ls doc/adr
0001-something-about-node-js.adoc
0002-slash-slash-slash.adoc
0003-bar.adoc
